tests/phpunit/CRM/MembershipExtras/Service/MembershipInstalmentsScheduleTest.php

```diff
@@ -644,6 +644,30 @@ public function testExceptionIsThrownIfMembershipTypeDurationUnitIsDay() {
     $this->getMembershipInstalmentsSchedule([$membershipType], MembershipInstalmentsSchedule::ANNUAL);
   }
 
+  /**
+   * Tests that instalments date are calculated properly and months follows a sequential order.
+   */
+  public function testRollingInstalmentDatesAreCalculatedProperly() {
```

Comment:
the name of the test should reflect the edge case that you are testing (which is that start date day is > 28).